libs/features/mas/src/variants/plans.js

```diff
@@ -179,5 +187,13 @@ export class Plans extends VariantLayout {
         line-height: 21px;
         padding: 2px 10px 3px;
     }
+    
+    :host([variant='plans'][segment='students']) {
```

Comment:
do we still need `segment='students'`  ? should we simply use `host([variant='plans-edu']` ?

Reply:
@yesil  With `host([variant='plans-edu']` none of those styles for  `merch-card[variant="plans"]` will not be applied (and we need them for students card as well) so I either need to do add `segment='students'` attribute to the card or I would need to add `merch-card[variant="plans-edu"]` to every single CSS selector `merch-card[variant="plans"]` which is not a solution really.

Reply:
what if we change them from `:host([variant='plans'])`  to : `:host([variant^='plans'])`

Reply:
Good idea! Done with `merch-card[variant^="plans"]` and `merch-card[variant="plans-edu"]`
No more segment attribute.